### PR TITLE
#Issue 13146: Unable to determine raw data structure for data ingestion

### PR DESCRIPTION
**Pull request by PhilTran01**
See sample output here:
https://redmine.oceanobservatories.org/issues/13146#note-20

### CODE REVIEW

File: find_raw_cabled_files.py [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+    """
+    Only get cabled assemblies.
+    Table provided by the cabled_drivers_list.txt file.
+    """ 
+    main_dir = '/rsn_cabled/rsn_data/DVT_Data'
+    directory = '/'.join([main_dir, refdes.split('-')[1].lower(), ''])
+
+    if os.path.isdir(directory):
+        return directory
+
```
[the remark this answers is not in the document]
Comment (PhilTran01) [reply find_raw_cabled_files.py]: I added a separate check in main on line 79.

[the remark this answers is not in the document]
Comment (danmergens) [reply find_raw_cabled_files.py]: But you need something in this function. If directory is not returned, nothing is returned. So you need to either return `None` after the `if` or not have the check in this function.

File: find_raw_cabled_files.py [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+                    print "File glob: " + file_path
+
+                    # Option to list out all the files available
+                    if arg['--list_files']:
+                        for path in glob_directory:
+                            print path
+                else:
+                    print "There are no files available." 
+            else:
+                print "This is not a valid cabled directory."
```
Comment (danmergens) [inline find_raw_cabled_files.py]: I think the statement needs to be more helpful to the user:

```python
print 'Unable to resolve directory for: ', ref_des
```

File: find_raw_cabled_files.py [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+        else:
+            print "This is not a valid cabled reference designator."
```
Comment (danmergens) [inline find_raw_cabled_files.py]: What is not an valid reference designator? Don't hide information from the user:

```python
print 'Invalid reference designator: ', refdes
```

File: find_raw_cabled_files.py [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+                    print "Reader Type(s):",
+                    print ', '.join(reader_type)        
+                    print "File glob: " + file_path
+
+                    # Option to list out all the files available
+                    if arg['--list_files']:
+                        for path in glob_directory:
+                            print path
+                else:
+                    print "There are no files available."
```
Comment (danmergens) [inline find_raw_cabled_files.py]:
```python
print 'Unable to find files to ingest for ', refdes
```

**Pull request merged**